ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/patch-panel.j2

```diff
@@ -28,7 +28,19 @@
 | {{ patch.name }} | {{ patch_panel_patch.enabled }} | {{ patch_panel_patch.connector_a_type }} | {{ patch_panel_patch.connector_a_endpoint }} | {{ patch_panel_patch.connector_b_type }} | {{ patch_panel_patch.connector_b_endpoint }} |
 {%         endif %}
 {%     endfor %}
+{%     if patch_panel.connector.interface.recovery.review_delay.min is arista.avd.defined %}
```

Comment:
Let's move these above the table (right after the `### Patch Panel Summary` heading and then insert a heading after (above the table) with `#### Patch Panel Connections`